src/pages/wordbookPage/WordBook.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Header from '../../components/Header.jsx';
+import '../../styles/wordbookStyles/WordBook.scss';
+import TabBarItem from '../../assets/TabBarItem.svg';
+import Line from '../../assets/Line.svg';
+import { Link } from 'react-router-dom';
+import DateData from './DateData.js';
+
+const WordBook = () => {
+    return (
+        <div className="wordbook-container">
+            <div className="wordbook-container">
+                <Header isNavigationBar={false} viewName="ViewName" />
+                <div className="date-section">
+                    {DateData.map((content, index) => (
+                        <Link
+                            to={`/WordWeek${index}`}
+                            key={index}
+                            className="dateitem-container"
+                        >
+                            <div className="date-item">
+                                <div className="date-info">
+                                    <span className="date-text">
+                                        {content.dateText}
+                                    </span>
+                                    <span className="date-range">
+                                        {content.dateRange}
+                                    </span>
+                                </div>
+                                <img
+                                    className="tabbar-item"
+                                    src={TabBarItem}
+                                    alt="front"
+                                />
+                            </div>
+                            <img className="line-word" src={Line} alt="Line" />
```

Comment:
component 폴더 내에 Divider.jsx 컴포넌트가 존재합니다! 
```suggestion
                            <Divier />
```
```jsx
import '../../components/Divider';
```
임포트해서 사용해주시면 될 것 같습니다